src/eval.c

```diff
@@ -7113,6 +7385,9 @@ get_tv_string_buf_chk(typval_T *varp, char_u *buf)
 	case VAR_SPECIAL:
 	    STRCPY(buf, get_var_special_name(varp->vval.v_number));
 	    return buf;
+        case VAR_BLOB:
+	    EMSG(_("E964: using Blob as a String"));
```

Comment:
The wording differ to E963 above.
This should be "E964: Using a Blob as a String".
(E963 has been already used by another patch recently merged. So the error numbers should be updated when merging.)

Reply:
Thanks yoru review. There are some wording using lower letter in few lines above.

https://github.com/vim/vim/pull/3638/files/324aca2e5df08652dccecfc3dbaeaa7115341502#diff-de9ab8906fef668ed8042046e0a1eb02R7374

@brammool Can I update them too?